Makefile

```diff
@@ -226,7 +226,7 @@ flake-unit-trace:
 
 flake-unit-race:
 	@$(call print, "Flake hunting races in unit tests.")
-	while [ $$? -eq 0 ]; do env CGO_ENABLED=1 GORACE="history_size=7 halt_on_errors=1" $(GOLIST) | $(XARGS) env $(GOTEST) -race -test.timeout=20m -count=1; done
+	while [ $$? -eq 0 ]; do $(RACE_PKG) | $(XARGS) env CGO_ENABLED=1 GORACE="history_size=7 halt_on_errors=1" $(GOTEST) -race -test.timeout=20m -count=1; done
```

Comment:
The variable name `RACE_PKG` threw me off here... Since it doesn't actually contain the package(s) but the command to list them. Should we rename it perhaps?

Reply:
IIUC it's a separate name / var since we replace that list with one entry when we specify the package?

So we have the listing from `GOLIST` duplicated in `RACE_PKG` and `COVER_PKG`.

Reply:
Yeah, and we don't use `RACE_PKG` or `COVER_PKG` anywhere, nor do we have those variables in `lnd`. So I'm not sure why we have them. So I thought, let's repurpose one of them.

On second thought, what would be the better approach is to adjust `GOLIST` here (similar to `RACE_PKG`):

https://github.com/lightninglabs/taproot-assets/blob/9e92b52aef3362cbc1ddd9ccea7b7e82e719ff57/make/testing_flags.mk#L19-L25

Reply:
But `GOLIST` does also contains the *command* to list the packages. It's weird, because the assignment is with `:=`, but if I run the Makefile target I see this in the cli:

```
gijs@gijs-Legion:~/git/taproot-assets$ make flake-unit-race
 Flake hunting races in unit tests.
while [ $? -eq 0 ]; do go list -deps -tags="dev monitoring" ./... | grep 'github.com/lightninglabs/taproot-assets' | xargs -L 1 env CGO_ENABLED=1 GORACE="history_size=7 halt_on_errors=1" GOEXPERIMENT=loopvar GO111MODULE=on go test -race -test.timeout=20m -count=1; done
```

`:=` doesn't seem to do what I think it does.